src/providers/journal/journal.ts

```diff
@@ -1,14 +1,25 @@
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { AppConfigProvider } from '../app-config/app-config';
+import moment from 'moment';
 
 @Injectable()
 export class JournalProvider {
 
   constructor(public http: HttpClient, public appConfig: AppConfigProvider) {}
 
-  getJournal(){
-    return this.http.get(this.appConfig.getAppConfig().journal.journalUrl);
+  getJournal(lastRefreshed: Date){
+    let get$;
+    if (lastRefreshed === null) {
+      get$ = this.http.get(this.appConfig.getAppConfig().journal.journalUrl);
```

Comment:
Could we return the response here and drop the else clause?

Reply:
Done.